templates/services/merge/consensus/lighthouse.tmpl

```diff
@@ -21,7 +27,8 @@
     expose:
       - {{.ClApiPort}}     
     command:
-      - lighthouse
+      - lighthouse{{if .CcRemoteDpl}}
```

Comment:
Why on `CcRemoteDpl` and not on `CcCustomCfg`?

Reply:
Yes, will fix it.